### DISCUSSION TITLE
Can I get a consolidated Report from various platforms?

### DISCUSSION THREAD

**Discussion by sriram-subramanianpr** [allure-framework/allure2#2136] [General Discussion]
Today, I have Four different automation solutions involving APPIUM (WDIO), API (REST ASSURE), WALDO( NO CODE), and DETOX(JS) I can generate seperate Allure reports for each of them, but can I get a consolidated test report for all my automation needs, Does Allure support it?

**Comment by baev** [comment]
Allure supports aggregated report generation. You can achieve so by providing multiple results folders (`allure-results`) to the report generation command, e.g. `allure generate appium-tests/allure-results api-tests/allure-results`.

Besides that, you may want to check out [Allure TestOps](https://qameta.io) — the platform that consolidates all the testing results in one place
